demo.py

```diff
@@ -50,32 +53,38 @@ def get_qa_model():
             passage_file=args.passage_file,
             device=args.device,
         )
+
     return model
 
 
 qa_model = get_qa_model()
 
+# Write the page title.
 st.title(":robot_face: Question Answering Machine :robot_face:")
 
-default_question = "Who won the first Nobel Prize in Physics?"
-
+# Write the question input form.
 with st.form("question_form"):
-    question = st.text_input("Input a question:", value=default_question, max_chars=200)
+    question = st.text_input("Input a question:", value=DEFAULT_QUESTION, max_chars=200)
     submitted = st.form_submit_button("Go")
 
 if submitted and len(question) > 0:
+    # Write the question.
     st.header("Question")
     st.write(question)
 
+    # Write the answer candidates.
     st.header("Answer Candidates")
+    # Fetch the answer candidates.
     with st.spinner("Computing..."):
-        answer_candidates = qa_model.answer_question(question, num_reading_passages=3)
+        answer_candidates = qa_model.answer_question(question, num_passages_to_read=3)
 
     for i, answer_candidate in enumerate(answer_candidates):
+        input_text = answer_candidate.input_text
         answer_text = answer_candidate.answer_text
-        passage_text = answer_candidate.passage_text
+        answer_start, answer_end = answer_candidate.answer_text_span
```

Comment:
Get the span of the extracted answer so that the answer is highlighted in the passage text.